venues/ICLR.cc/2018/Conference/python/review-revisions.py

```diff
@@ -0,0 +1,60 @@
+import openreview
+import argparse
+import config
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--username')
+parser.add_argument('--password')
+parser.add_argument('--baseurl', help = "base URL")
+args = parser.parse_args()
+
+client = openreview.Client(username=args.username, password=args.password, baseurl=args.baseurl)
+
+def create_revision_invitation(forum, referent, signature):
+    params = {
+        'writers': [config.CONF],
+        'signatures': [config.CONF],
+        'readers': ['everyone'],
+        'invitees': [signature],
+        'reply': {
+            'forum': forum,
+            'referent': referent,
+            'writers': {
+                'values': [config.CONF]
+            },
+            'signatures': {
+                'values-regex': signature + '|ICLR.cc/2018/Conference'
+            },
+            'readers': {
+                'values':['everyone']
+            },
+            'content': config.official_review_params['reply']['content']
+        }
+    }
+
+    signature_components = signature.split('/')
+    paper_num = signature_components[3]
+    anonreviewer = signature_components[4]
+
+    revise_review = openreview.Invitation('ICLR.cc/2018/Conference/-/{0}/{1}/Revise_Review'.format(anonreviewer, paper_num), **params)
+    return revise_review
+
+review_invitations = client.get_invitations(regex = config.CONF + '/-/Paper.*/Official_Review')
+reviews = client.get_notes(invitation = config.CONF + '/-/Paper.*/Official_Review')
+
+for inv in review_invitations:
+    forum_reviews = [r for r in reviews if r.forum == inv.reply['forum']]
+
+    inv.reply['writers'] = {'values': []}
```

Comment:
the invitation reply writers gets set to empty here, which prevents reviewers from directly editing any future reviews that are posted